operations/app/terraform/modules/function_app/apps.tf

```diff
@@ -40,5 +40,15 @@ module "pdi_function_app" {
     "${each.value.mi_accountName}"        = each.value.mi_accountValue
     fhir_url                              = each.value.fhir_url
     "AzureWebJobs.convertToFhir.Disabled" = each.value.AzureWebJobs_convertToFhir_Disabled
```

Comment:
I see the value this references changed.  I'm not sure where to find it in the function app...